modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -83,6 +83,8 @@ import {{invokerPackage}}.auth.OAuth;
  */
 {{>generatedAnnotation}}
 public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
+  public static final String VERSION = "{{{artifactVersion}}}";
```

Comment:
What about putting it in Configuration.java instead? 

I think that's how we put it in the C#, Ruby clients

Reply:
Thanks! I'll take a look at how it looks in C# and Ruby and I'll update this PR to do something similar.

Reply:
👍 

Here is an example for the C# client: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/csharp-netcore/Configuration.mustache#L33

Reply:
Moved to `Configuration` instead.

Reply:
I realized after I pushed that now I could remove `[jersey]` from commit message and PR title, since almost all Java-based clients use the same `Configuration.mustache`.
Want me to redo and force push?

Reply:
I added this to the `vertx` client as well, so now it's for all Java clients (AFAICT).
Removed `[jersey]` from commit message and PR title.